### ISSUE TITLE
Don't show older version for download

### ISSUE THREAD

**Issue by pkevan** [WordPress/wporg-main-2022#480]
**Describe the bug**
Currently when a release is live, we build a locale based download, which unless present we end up with an older version linked for download and in the release pages.

Examples:

![Screenshot 2024-07-18 at 11 00 59](https://github.com/user-attachments/assets/a78ea0a8-27c1-4a7c-8daf-2c73b2ac0d1f)
![Screenshot 2024-07-18 at 11 01 16](https://github.com/user-attachments/assets/e394f569-6e79-4909-9bef-72bc6db9f08e)
![Screenshot 2024-07-18 at 11 01 38](https://github.com/user-attachments/assets/9e35a6c5-f4c0-4f51-b252-e1ee88f7a8e7)
![Screenshot 2024-07-18 at 11 01 50](https://github.com/user-attachments/assets/ecb630fd-b505-4cc4-b00f-0ff7e052259b)

urls: https://af.wordpress.org/download/ and https://hu.wordpress.org/download/

**Expected behavior**
Once a new release is available, we should fallback to the default version rather than show older versions available for download.

On the releases page, we should show the latest releases and point releases, irrelevant of whether a local exists, and only show locale downloads for the versions which are built.

As a bonus, we could improve the Rosetta release manager dashboard to highlight that a locale needs building, or what might be blocking it from doing so, as it's often a question that gets asked.

**Comment by tobifjellner** [comment]
In [our Polyglots handbook page https://make.wordpress.org/polyglots/handbook/for-editors/packaging-localized-wordpress/automated-release-packages/](https://make.wordpress.org/polyglots/handbook/for-editors/packaging-localized-wordpress/automated-release-packages/) you can when a language pack and/or a complete localized installation pack get generated.

Unfortunately, the Afrikaans locale team is currently dormant, and way below the thresholds to get fresh language packs. See https://translate.wordpress.org/stats/

In addition, this week we've had a temporary technical issue where the installation packs don't get generated even for locales that would be eligible.

I see no reason to hide the archive of some earlier releases just because some specific releases are missing.

But it could be useful to specify that this archive only contains localized versions, and that the full WordPress version history can be found on (link)the global site(/link).

**Comment by ryelle** [comment]
I wonder if this was related to the locale package issues, but when I go to https://af.wordpress.org/download/ and click the download button, I get `https://wordpress.org/latest.zip`, which is 6.6 (default English). So that seems okay to me.

The release table is still showing 5.7.11 as the latest release though. I do think we could merge in the main release list and flag that the merged releases are English? Or maybe just update the "Latest release" section with the real latest release, noting that it's not translated? Something like: 

<img width="845" alt="Screenshot 2024-07-18 at 3 10 24 PM" src="https://github.com/user-attachments/assets/8be43f4f-f7fc-427e-a08a-396f0cb6e9b2">

**Comment by pkevan** [comment]
> but when I go to https://af.wordpress.org/download/ and click the download button, I get https://wordpress.org/latest.zip, which is 6.6 (default English). So that seems okay to me.

Yes - i was crossing two issues here to highlight inconsistencies, but agree with your assessment around the releases page, particularly when we reference the latest version as being preferred.
